src/app/views/admin-views/profile-module/components/profile-page/profile.component.ts

```diff
@@ -36,6 +39,13 @@ export class ProfileComponent  extends BaseClass implements OnInit{
     }
 
   ngOnInit() {
+    this.commonRequestService.request(RequestEnums.GET_GENDER_TYPES).subscribe(response=>{
```

Comment:
Write in seperate function and call it inside OnInit

Reply:
Done